src/routes/management-api/users.ts

```diff
@@ -133,6 +133,12 @@ export class UsersMgmtController extends Controller {
       throw new HTTPException(404);
     }
 
+    if (!!user.linked_to) {
```

Comment:
It would work just a well with `if(user.linked_to)` but.. maybe this is clearer?

Reply:
It's literally the same I just like writing it and invent reasons like it being more explicit because it shows I haven't forgotten something

I can add an eslint rule to stop me doing this if you hate it

I saw on coding golf that we can use a bitwise operator for something similar so I can start throwing that 😅

Reply:
```suggestion
    if (user.linked_to) {
```


You make a point where this is harder to read. When the eyes see an exclamation mark they think "negation"